### ISSUE TITLE
`jsdoc/no-bad-blocks` doesn't report if the comment block has a description

### ISSUE THREAD

**Issue by alexilyaev** [gajus/eslint-plugin-jsdoc#642]
## Expected behavior

`jsdoc/no-bad-blocks` should report bad JSDoc block for a block comment that start with `/*` instead of `/**` and has a description, followed by JSDoc tags (e.g. `@param`)

## Actual behavior

The rule doesn't report if the comment block starts with a description.  
It only reports if the first line is a JSDoc tag.

## ESLint Config

```js
module.exports = {
  plugins: ['jsdoc'],

  parser: 'babel-eslint',
  parserOptions: {
    ecmaVersion: 2020,
    sourceType: 'script',
  },

  rules: {
    'jsdoc/no-bad-blocks': 1,
  },
};
```

## ESLint sample

```js
/*
 * Some description.
 *
 * @returns {string} Some string
 */
function echo() {
  return 'Something';
}
```

## Environment

- Node version: v14.9.0
- ESLint version v7.9.0
- `eslint-plugin-jsdoc` version: 30.5.1

**Comment by brettz9** [comment]
Forgot to link the commit to the issue, but this is now fixed in `master` so closing: 9a568638f59960c8008fc2d03e4d83252a1e3072

Thanks for the report!

**Comment by brettz9** [comment]
Also released in [v30.5.3](https://github.com/gajus/eslint-plugin-jsdoc/releases/tag/v30.5.3).
